storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options that are not possible with in-tree volume plug-ins, such as volume snapshots.
```

Comment:
A little more clear if the example "volume snapshots" is closer to the thing it is an example of (storage options, not in-tree volume plug-ins):
`CSI Operators give {product-title} users storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins.`

Reply:
This is great! Clears up any confusion that it's `storage options -> snapshots`, and not `volume plug-ins -> snapshots`.